docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst

```diff
@@ -60,17 +60,17 @@ Selecting a Trace Output Format
 
 FireSim supports three trace output formats, which can be set in your
 ``config_runtime.yaml`` file with the ``output_format`` option in the
-``[tracing]`` section:
+``tracing`` section:
 
 .. code-block:: ini
 
-   [tracing]
-   enable=yes
+   tracing:
```

Comment:
I guess there are many examples of this :(. don't let the DOCREF stuff block the PR then

Reply:
Yea. I'll leave this to future work.